doc/api/util.md

```diff
@@ -943,11 +1450,11 @@ added: v0.6.0
 deprecated: v4.0.0
 -->
 
-> Stability: 0 - Deprecated
+> Stability: 0 - Deprecated: Use [`Array.isArray`][] instead.
```

Comment:
Maybe `Array.isArray()` as per our style guide note about parentheses after methods (+ below and in the bottom reference).

Reply:
@vsemozhetbyt Yup, done!